sdk/spring/CHANGELOG.md

```diff
@@ -21,6 +21,14 @@ This section includes changes in `spring-cloud-azure-autoconfigure` module.
 #### Dependency Updates
 - Upgrade spring-security to 5.6.4 to address [CVE-2022-22978](https://spring.io/blog/2022/05/15/cve-2022-22978-authorization-bypass-in-regexrequestmatcher) [#29304](https://github.com/Azure/azure-sdk-for-java/pull/29304).
 
+### spring-cloud-azure-starter-servicebus-jms
```

Comment:
The jms is part of the Spring Cloud Azure Autoconfigure

Reply:
updated